.github/workflows/ci.yml

```diff
@@ -3,16 +3,22 @@ name: CI
 on:
   pull_request:
   push:
+  schedule:
+    - cron: '0 0/2 * * *'
```

Comment:
```suggestion
```

There is no reason to heat the earth for this.

Reply:
I changed the cron to run on once per day only. The CI runtime is about a minute, so the total energy consumption is minimal.

Reply:
I refuse to merge any configuration that contains a cron schedule.

Reply:
Commited.